flow/record/tools/rdump.py

```diff
@@ -195,10 +209,30 @@ def main(argv: list[str] | None = None) -> int:
 
     args = parser.parse_args(argv)
 
-    levels = [logging.WARNING, logging.INFO, logging.DEBUG]
+    levels = [logging.WARNING, logging.INFO, logging.DEBUG, LOGGING_TRACE_LEVEL]
     level = levels[min(len(levels) - 1, args.verbose)]
     logging.basicConfig(level=level, format="%(asctime)s %(levelname)s %(message)s")
 
+    if HAS_STRUCTLOG:
+        # We have structlog, configure Python logging to use it for rendering
+        console_renderer = structlog.dev.ConsoleRenderer()
+        handler = logging.StreamHandler()
+        handler.setFormatter(
+            structlog.stdlib.ProcessorFormatter(
+                processor=console_renderer,
+                foreign_pre_chain=[
+                    structlog.stdlib.add_logger_name,
+                    structlog.stdlib.add_log_level,
+                    structlog.processors.TimeStamper(fmt="iso"),
+                ],
+            )
+        )
+
+        # Clear existing handlers and add our structlog handler
```

Comment:
Don't know if this will be necessary in flow.record, but I had to change this logic in Dissect to check for conflicting log handlers: https://github.com/fox-it/dissect.target/blob/9abc53c5f1a4ea032751fff554a8f22ab5ceb2f3/dissect/target/tools/logging.py#L111

If it's not necessary then this is fine, but maybe worth a double check.

Reply:
I don't think it will be a problem, we clear all handlers using:

```
root_logger.handlers.clear()
```

only thing is that we don't call `.close()` on the existing handlers, but I don't think this will cause any issues.